oclint-scripts/oclintscripts/path.py

```diff
@@ -73,8 +73,8 @@ class url:
     json_compilation_database = 'https://github.com/oclint/oclint-json-compilation-database.git'
     xcodebuild = 'https://github.com/oclint/oclint-xcodebuild.git'
 
-    clang_prebuilt_binary_for_macos = 'http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-apple-darwin.tar.xz'
-    clang_prebuilt_binary_for_ubuntu_lts_16 = 'http://releases.llvm.org/8.0.0/clang+llvm-8.0.0-x86_64-linux-gnu-ubuntu-16.04.tar.xz'
+    clang_prebuilt_binary_for_macos = 'http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-darwin-apple.tar.xz'
+    clang_prebuilt_binary_for_ubuntu = 'http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-14.04.tar.xz'
```

Comment:
I second your renaming of this variable, but if you do a global search, you will find that two other places (if I remember correctly) use this variable, where you also need to update.

Reply:
Oh！It's my fault，I just want to change the value inside，not the variable name.
I'll change it right away。